pkg/osbuild/osbuild-exec.go

```diff
@@ -27,9 +27,10 @@ const (
 var OSBuildCmd = "osbuild"
 
 type OSBuildOptions struct {
-	StoreDir  string
-	OutputDir string
-	ExtraEnv  []string
+	StoreDir    string
+	OutputDir   string
+	Checkpoints []string
+	ExtraEnv    []string
```

Comment:
Move exports here too?  I like the idea of all the osbuild args being bundled together and only having function args for things that aren't directly passed to the osbuild call (if we ever add any).